app/controllers/flows_controller.rb

```diff
@@ -70,7 +71,7 @@ def show
 
   def current_intake
     if %w[state_file_az state_file_ny].include?(params[:id] || params[:type])
-      GlobalID.find(session[:state_file_intake])
+      current_state_file_az_intake || current_state_file_ny_intake
```

Comment:
this makes me wonder: can we configure devise to only allow one of these to be signed in at the same time? (my assumption right now is that both can be, which i assume we don't want?)

Reply:
Done!